### ISSUE TITLE
Salvataggio firma tecnico

### ISSUE THREAD

**Issue by Dasc3er** [devcode-it/openstamanager#801]
## Comportamento richiesto

Per le Anagrafiche di tipo Tecnico, può essere utile introdurre la possibilità di caricare come allegato al firma personale da utilizzare poi in automatico per la funzione "Anteprima e firma" delle **Attività** e nella stampa del rapportino d'intervento

**Comment by lucasalva87** [comment]
Esatto, possiamo attivare un nome specifico per l'allegato esattamente come avviene per il logo azienda. Es. "Firma". Terrei un nome generico perché potrebbe tornare utile prevederla anche per l'Azienda. Es di documenti prodotti che devono essere inviati con firma del legale rappresentate.
